pom.xml

```diff
@@ -9,23 +11,14 @@
   </parent>
   <groupId>com.solace.quarkus</groupId>
   <artifactId>quarkus-solace-parent</artifactId>
-  <version>999-SNAPSHOT</version>
+  <version>0.0.1-SNAPSHOT</version>
```

Comment:
@ozangunalp FYI.

Reply:
Check out the releasing.md doc to see what the commits on main will be like. The intention is that main would always be one patch number higher than the last released version. I didn't have to pick this version but I figured it would be overwritten soon whenever this is released.

But we can also use release branches if you'd prefer. It currently has some support for this but we can automate the branch creation too if this is the only way you intend to develop. That way main and all features branches can stay as `999.0.0-SNAPSHOT`, then only the release branches will have proper versions.

Reply:
When you go to release from main, the action will first make a commit where the version is set to the release version. Then it will build / deploy / tag it. Then it will increment the patch number and add snapshot again, so after a 1.0.0 release from main, main would be 1.0.1-SNAPSHOT.

Then if you select auto version for the next release from main, you wouldn't need to enter 1.0.1. the default auto version will select that for you, or if it was a breaking change / new feature you could specify the new release version manually, like 1.1.0 instead

Reply:
Okay, so that means a commit is always triggered to update all the pom's during the release process?

Reply:
Yes, two commits are pushed on whatever branch it is released from
One to set to the desired version, which is what we deploy (eg `1.0.0`)
and then one more that increments the patch version and adds -SNAPSHOT (eg `1.0.1-SNAPSHOT`).

Reply:
Okay. This will update only the modules included as part of release or complete project? My only concern is that pom is updated for every release we trigger. How much time do you think it will take if we switch to release branch process.